cas-server-support-duo/src/main/java/com/duosecurity/duoweb/Util.java

```diff
@@ -7,6 +7,15 @@
 import javax.crypto.spec.SecretKeySpec;
 
 public class Util {
+    /**
+     * TODO: 12/10/15   move to DuoWeb
```

Comment:
That makes sense. I'll see what I can do to lean on Duo there too, or replace Duo with another TOPT implementation...